setup.cfg

```diff
@@ -30,6 +30,7 @@ project_urls =
     Funding = https://www.djangoproject.com/fundraising/
     Source = https://github.com/django/django
     Tracker = https://code.djangoproject.com/
+    Release notes = https://docs.djangoproject.com/en/3.0/releases/
```

Comment:
Let's always point at the latest stable release. This will redirect to the appropriate version automatically:
```
    Release Notes = https://docs.djangoproject.com/en/stable/releases/
```

Reply:
Do we also want to move this up underneath `Documentation`?

Reply:
Pointed to stable, moved anyway, amended & force-pushed.